.ci/README.md

```diff
@@ -32,7 +32,16 @@ This will give you a command line interface which is rich and will allow you to
 If you develop MagicModules (code generation features), here are the things you'll want to know.
 
 ## Deploying the pipeline
-The pipeline config is generated using jinja2.  You'll want `j2cli`: `pip install j2cli`.  To generate the pipeline, use `j2 .ci/ci.yml.tmpl`.  To deploy it, use `fly -t sunrise sp -c <(j2 ci.yml.tmpl) -p magician`.
+The pipeline config is generated using jinja2.  To generate it;
+
+* Install `j2cli` with `pip install j2cli`
+* Enter the `.ci` directory; if you're in the repo root, that's with `cd .ci`
+* Test generating the pipeline with `j2 ci.yml.tmpl`
+  * Invocations from outside `.ci` won't work, `j2` will fail trying to import a dependency
+  
+Finally, if you'd like to deploy it:
+
+* To deploy it, use `fly -t sunrise sp -c <(j2 ci.yml.tmpl) -p magician` 
```

Comment:
Repeated wording here, mentioning `deploy it` on the line above and here.

But not a big deal